istresearch/traptor
# PULSE-3561 Make Traptor internal rule match stats collection configurable

## What's New?

An environment variable, ENABLE_STATS_COLLECTION, has been added  to allow users to disable  rule match statistics from being sent to redis. 

## How To Test

- Pull latest PULSE-3561 branch
- Add a keyword rule and test it, confirm that the rule match stats are being collected by default
- Add ENABLE_STATS_COLLECTION=true to your traptor.env or doctor-compose files 
  - Retest with your rule and confirm the stats are still collected
- Set ENABLE_STATS_COLLECTION to false (or any non-true)
  - Retest with your rule and confirm the stats are no longer being collected
